generate/beat/{{cookiecutter.beat}}/dev-tools/README.md

```diff
@@ -0,0 +1,10 @@
+Available scripts
```

Comment:
Do we really need these files as part of the generator? We just removed all dev-tools files :-) Can't we just link to the existing one and adjust the phrasing so the beat name itself is not needed and just reference to the `beat`?